### PR TITLE
Fix wall not opening from right <bug>

### PR DESCRIPTION
**Pull request by cbilstra**
Important bug for wall not opening and some checksytle improvements. Bug is fixed on line 252 of collisionresolver

### CODE REVIEW

File: src/main/java/doob/level/CollisionResolver.java [hunk 1]
```diff
@@ -242,7 +249,7 @@ private void handleDisappearingBall(Ball disappearingBall) {
 			dLog.info("All balls gone", DLog.Type.STATE);
 		} else if (walls.t0 == level.getLeft()) {
 			walls.t1.setOpen(true);
-		} else if (walls.t1 == level.getLeft()) {
+		} else if (walls.t1 == level.getRight()) {
```
Comment (jorisoudejans) [inline src/main/java/doob/level/CollisionResolver.java]: Ah yes, great! stupid typo...

**Comment by jorisoudejans** [comment]
Maybe this branch is an opportunity to fix all Checkstyle warnings in the project?

**Comment by cbilstra** [comment]
I think the name of the branch should be different in that occasion

**Comment by hlycklama** [comment]
Good work, are you sure it was a typo?

**Comment by jorisoudejans** [comment]
Yeah inconsistent if-else branch...

**Comment by jorisoudejans** [comment]
Okay, so are you opening another branch for Checkstyle?

**Comment by cbilstra** [comment]
Yeah eventually ;)

**Pull request merged**